### PR TITLE
sdk/dotnet-v0_21

### PR DESCRIPTION
**Pull request by SpiralOSS**
- everything adjusted for v0.21
- updates for the Falco.Datastar library (not complete, waiting on modifiers to settle)

### CODE REVIEW

**Comment by SpiralOSS** [comment]
@bencroker

**Comment by delaneyj** [comment]
need to pull in `develop` and do a `task build -f` to resolve the merge

**Comment by SpiralOSS** [comment]
@delaneyj @bencroker Done

**Review by delaneyj — APPROVED** [review]
@bencroker i'm ready for this to be in v0.21.1

File: examples/dotnet/CsharpAspServer/Program.cs [hunk 1]
```diff
@@ -45,33 +45,28 @@ public static void Main(string[] args)
...
         });
-        app.MapGet("/language/{lang:required}", (string lang, IServerSentEventService sseService) => sseService.MergeFragments($"""<span id="language">{lang}</span>"""));
-        app.MapGet("/patch", async (IServerSentEventService sseService, IDatastarSignals signals) =>
+        app.MapGet("/title/{title:required}", (string title, IServerSentEventService sseService) => sseService.MergeFragments($"""<span id="title">C# + {title}</span>"""));
+        app.MapGet("/check_more_examples", () => Task.CompletedTask); // do nothing
+        app.MapGet("/patch", async (IServerSentEventService sseService, ISignals dsStore) =>
         {
-            DataSignals signals = (signals as DataSignals) ?? throw new InvalidCastException("Unknown DataSignals passed");
-            DataSignals mergeSignals = new() { Output = $"Patched Output: {signals.Input}" };
+            Signals signals = (dsStore as Signals) ?? throw new InvalidCastException("Unknown Datastore passed");
```
Comment (bencroker) [inline examples/dotnet/CsharpAspServer/Program.cs]: Should `dsStore` and `Datastore` be renamed? We no longer use `store` anywhere in the library or docs.

Comment (SpiralOSS) [reply examples/dotnet/CsharpAspServer/Program.cs]: Thanks for catching. Will take a look.

Comment (SpiralOSS) [reply examples/dotnet/CsharpAspServer/Program.cs:52]: Thanks for catching. Will take a look.

Comment (SpiralOSS) [reply examples/dotnet/CsharpAspServer/Program.cs]: Corrected, didn't see any other "store"s.

**Comment by bencroker** [comment]
Merged, thanks!

**Pull request merged**